### ISSUE TITLE
Dynamically add custom logging

### ISSUE THREAD

**Issue by leeuw73** [frankframework/frankframework#1805]
In the IJA_API I have added a feature to make it possible to view logging of HTTP requests and responses. It's very helpful when there is an issue and you want to see for HTTP traffic what is received and sent.

In the IJA_API there is also HttpRequest logging. Also this feature is very helpful. In the IJA_API this is regulated by a separate pipe (which has to be included in the configuration). An improvement would be to add it to WebServiceListener/RestListener/ApiListener.

![afbeelding](https://user-images.githubusercontent.com/3647523/118239859-956f6180-b49a-11eb-8582-f12e6a7bf914.png)

![afbeelding](https://user-images.githubusercontent.com/3647523/118240506-61487080-b49b-11eb-8643-2b9c7a7ebbfa.png)

**Comment by jacodg** [comment]
Nice feature. We will do this in a more generic way so you can configure logging more fine grained on a separate page in the GUI

**Comment by nielsm5** [comment]
Semi-fixed in #1847. You will still need to manually configure (in the log4j4ibis.xml) which loggers/appenders you would like to use. The log level can be changed dynamically on the `log settings` page.

**Comment by nielsm5** [comment]
Extending the log is already possible, see #6142.
Spring boot like log config can be used to easily add new loggers through environment variables.
eq;
```
logging.level.root=INFO
logging.level.org.frankframework=DEBUG
logging.level.org.springframework=WARN
```
